src/test/Spec.hs

```diff
@@ -236,7 +236,7 @@ runContract fp n c =
     cs  <- Echidna.Solidity.contracts (fp NE.:| [])
     ads <- NE.toList <$> addresses
     let ads' = AbiAddress . addressWord160 <$> v ^. env . EVM.contracts . to keys
-    campaign (pure ()) v w ts (Just $ mkGenDict 0.15 (extractConstants cs ++ ads ++ ads') [] g (returnTypes cs))
+    campaign (pure ()) v w ts (Just $ mkGenDict 0.15 (extractConstants cs ++ ads ++ ads') [] g (returnTypes cs)) [[]]
```

Comment:
should this just be `[]` (no `Tx` list) and not `[[]]` (one `Tx` list containing no transactions)?

Reply:
Good catch. Fixed.